src/formatters/json/mod.rs

```diff
@@ -18,10 +17,9 @@ impl JsonFormatter {
 
 impl super::WriteEvent for JsonFormatter {
     fn write_event(&self, event: &Event<'static>, to: &mut Write) -> Result<(), Box<Error>> {
-        let template = try!(serde_json::to_string(event.message_template().text()));
         let isots = event.timestamp().format("%FT%T%.3fZ");
 
-        try!(write!(to, "{{\"@t\":\"{}\",\"@mt\":{}", isots, template));
+        try!(write!(to, "{{\"@t\":\"{}\",\"@mt\":\"{}\"", isots, event.message_template().text()));
```

Comment:
Even if the function is just a dummy that returns its input for now, perhaps we should insert the `json_escape()` calls we're going to need so that we can keep tabs on where they are required?